app/jobs/regular/export_csv_file.rb

```diff
@@ -221,6 +221,9 @@ def report_export
       if @extra[:group_id].present?
         @extra[:filters][:group] = @extra[:group_id].to_i
       end
+      if @extra[:include_subcategories].present?
+        @extra[:filters][:'include-subcategories'] = !!ActiveRecord::Type::Boolean.new.cast(@extra[:include_subcategories])
```

Comment:
Why are you using dash "`include-subcategories`" instead of underscore "`include_subcategories`" on the server-side?

Reply:
That is the name of the filter. The client uses 'include-subcategories' which is changed to 'include_subcategories' when a request is made and then the server renames this back.

I tried to follow what we have now, which is not the best. I will change it.